build.py

```diff
@@ -0,0 +1,33 @@
+import shutil
+from pathlib import Path
+
+from staticjinja import Site
+from yaml import Loader, load
+
+import sass
+
+def data():
+    data_dict = {}
+    yaml = Path("./data").glob("*.yaml")
+    data_dict = {file_path.stem: load(file_path.open(), Loader=Loader) for file_path in yaml}
+
+    return data_dict
+
+buildpath = Path('output')
+
+if buildpath.exists() and buildpath.is_dir():
+    shutil.rmtree(buildpath)
```

Comment:
This should probably be under the if-block below.

Reply:
again, great comment and I love the improvements that have come out of this review. I've tried to organize things into separate functions to make it more readable and modular. let me know what you think. cheers.

Reply:
Yeah, my main concern was that this happens at `import` time, which is a known bad practice.